pretyping/evarconv.ml

```diff
@@ -43,6 +57,16 @@ let _ = Goptions.declare_bool_option {
   Goptions.optwrite = (fun a -> debug_unification:=a);
 }
 
+let debug_ho_unification = ref (false)
+let _ = Goptions.declare_bool_option {
+  Goptions.optdepr = false;
+  Goptions.optname =
+    "Print debug information for the higer-order unification algorithm";
```

Comment:
Typo: higher.

Also, `optname` is used for printing a message on `Test`. It should take the place of `s` in `The s mode is off/on`. So, it should be something like `the higher-order unification algorithm
debug printing`.